config.example.json

```diff
@@ -1,7 +1,8 @@
 {
 	"botOwnerUserId": "",
 	"twitch": {
-		"clientId": ""
+		"clientId": "",
+		"accessToken": ""
```

Comment:
Not a fan of using an arbitrary accessToken. We can use Twitch's App access tokens to fully automate the tokens.

[App access tokens](https://dev.twitch.tv/docs/authentication#types-of-tokens)
[OAuth Client Credentials Flow](https://dev.twitch.tv/docs/authentication/getting-tokens-oauth#oauth-client-credentials-flow)